app/backend/src/couchers/migrations/versions/bf12729fa8eb_sms_verification_constraints.py

```diff
@@ -24,4 +24,4 @@ def upgrade():
 
 
 def downgrade():
-    raise hell
+    raise hell  # noqa: F821
```

Comment:
I mean we can change this to a real exception, just `raise Exception("hell")`

Reply:
I guess you're right - did not want to break Christian's joke :) Changed it now